internal/support/scheduler/application/interval.go

```diff
@@ -154,12 +155,17 @@ func LoadIntervalToSchedulerManager(dic *di.Container) errors.EdgeX {
 	// Load intervals from config to DB
 	configuration := container.ConfigurationFrom(dic.Get)
 	for i := range configuration.Intervals {
-		interval := models.Interval{
+		dto := dtos.Interval{
 			Name:     configuration.Intervals[i].Name,
 			Start:    configuration.Intervals[i].Start,
 			End:      configuration.Intervals[i].End,
 			Interval: configuration.Intervals[i].Interval,
 		}
+		validateErr := common.Validate(dto)
+		if validateErr != nil {
+			return errors.NewCommonEdgeXWrapper(validateErr)
```

Comment:
wrap the error with the proper error message:
`validate pre-defined Interval %s from configuration failed , dto.Name`

Reply:
Modified.